docs/changelogs/CHANGELOG-2.27.md

```diff
@@ -0,0 +1,241 @@
+
+- Add KubeVirt DS in the charts repo to generate images for the mirrored images command ([#14064](https://github.com/kubermatic/kubermatic/pull/14064))
+- Add `VolumeProvisioner` field in the InfraStorageClasses for the KubeVirt provider in the seed, to indicate whether the storage class can be used by the CSI or the CDI ([#14111](https://github.com/kubermatic/kubermatic/pull/14111))
+- Support `CSIDriverOperator` field in the seed object to customize csi driver images in the KubeVirt user cluster ([#14147](https://github.com/kubermatic/kubermatic/pull/14147))
+- Support KubeVirt VMs LiveMigrate as an eviction strategy ([#14076](https://github.com/kubermatic/kubermatic/pull/14076))
+- Bump KubeVirt csi-driver-operator to support zone-aware topologies ([#13833](https://github.com/kubermatic/kubermatic/pull/13833))
+- Bump the new KubeVirt csi controller operator and add supports to infra storage class labels ([#13827](https://github.com/kubermatic/kubermatic/pull/13827))
+- Include KubeVirt CCM and Fluent-Bit images in the mirror-images command ([#14063](https://github.com/kubermatic/kubermatic/pull/14063))
+- Remove storage classes filtration in KubeVirt Namespaced mode ([#13985](https://github.com/kubermatic/kubermatic/pull/13985))
+- Bump KubeVirt CSI Driver operator to v0.4.1 in KKP ([#14011](https://github.com/kubermatic/kubermatic/pull/14011))
+- Add support for KubeVirt CSI driver operator images overwrite ([#14142](https://github.com/kubermatic/kubermatic/pull/14142))
+- Add support for KubeVirt provider network ([#13791](https://github.com/kubermatic/kubermatic/pull/13791))
+- Setup KubeVirt network controller in the seed-controller-manager ([#13858](https://github.com/kubermatic/kubermatic/pull/13858))
+- Support KubeVirt VolumeBindingMode in the tenant storage class ([#13821](https://github.com/kubermatic/kubermatic/pull/13821))
+- Update kubevirt csi driver ([#13818](https://github.com/kubermatic/kubermatic/pull/13818))
+
+#### OpenStack
+
+- Remove redundant storage classes from OpenStack CSI addon ([#13920](https://github.com/kubermatic/kubermatic/pull/13920))
+- Upgrade cloud-provider-openstack for v1.30 and v1.31 ([#13899](https://github.com/kubermatic/kubermatic/pull/13899))
+- Mark `domain` as optional field for OpenStack preset ([#13948](https://github.com/kubermatic/kubermatic/pull/13948))
+- Add `NodePortsAllowedIPRanges` option on the seed datacenter level, giving an option to override default setting for all user clusters on OpenStack provider ([#14029](https://github.com/kubermatic/kubermatic/pull/14029))
+
+#### Packet
+
+- Use official Equinix SDK instead of deprecated packethost/packngo ([#13851](https://github.com/kubermatic/kubermatic/pull/13851))
+
+#### VSphere
+
+- Default datastore validation is skipped, if datastore cluster is specified for the user clusters on vSphere ([#13708](https://github.com/kubermatic/kubermatic/pull/13708))
+- TBD ([#14087](https://github.com/kubermatic/kubermatic/pull/14087))
```

Comment:
Pester whoever wrote this to come up with a real entry.

Reply:
FYI, checking with @soer3n  on this entry.

Reply:
done!